packages/modals/src/styled/StyledTooltipDialog.ts

```diff
@@ -23,14 +24,19 @@ export interface IStyledTooltipDialogProps
   transitionState?: TransitionStatus;
 }
 
+const sizeStyles = (props: ThemeProps<DefaultTheme>) => `
+  padding: ${props.theme.space.base * 5}px;
+  width: 400px;
+  &:has(${StyledTooltipDialogClose}) > :first-child {
```

Comment:
I'm so happy support for this selector is as high as it is. Didn't realize its been in safari since version 15!